js/modifyContactAdminButtonURL.js

```diff
@@ -3,15 +3,13 @@ $('document').ready(function() {
 
         if(searchResults.length === 1) {
             var button = searchResults[0];
-            var username = document.getElementById('username-reference').textContent
-            
-            button.href = contactAdminButtonSettings.url
-                + '&gatorlink=' + contactAdminButtonSettings.username 
-                + '&project_id=' + contactAdminButtonSettings.project_id
-                + '&redcap_username=' + contactAdminButtonSettings.username
-                + '&registered_in_redcap=1'
-                + '&first_name=' + contactAdminButtonSettings.user_firstname
-                + '&last_name=' + contactAdminButtonSettings.user_lastname
-                + '&email=' + contactAdminButtonSettings.email
+            var length = contactAdminButtonValuePairs.length;
+            button.href = contactAdminButtonURL;
+            button.href += '&registered_in_redcap=1'
```

Comment:
You don't want to add static parameters to the base URL. If someone needs a static parameter, they can added it to the Base URL in the module configuration.